c2corg_ui/templates/helpers/list.html

```diff
@@ -23,4 +23,15 @@
   <span class="glyphicon glyphicon-globe"></span>
   <span translate>Show map</span>
 </button>
+<button class="btn orange-btn btn-default no-list-btn hidden-xs"
+        ng-show="rowList" ng-click="rowList = false;">
+  <span class="glyphicon glyphicon-globe"></span>
+  <span translate>Show Card</span>
+</button>
+<button class="btn orange-btn btn-default no-list-btn hidden-xs"
+        ng-show="!rowList" ng-click="rowList = true">
+  <span class="glyphicon glyphicon-globe"></span>
+  <span translate>Show List</span>
+</button>
```

Comment:
Instead of 2 buttons you could have only one that would toggle the value of ``rowList``:
```
<button class="btn orange-btn btn-default no-list-btn hidden-xs" ng-init="rowList = false" ng-click="rowList = !rowList">
￼  <span class="glyphicon glyphicon-globe"></span>
￼  <span translate ng-show="!rowList">Show List</span>
  <span translate ng-show="rowList">Show Cards</span>
￼</button>
```